README.md

```diff
@@ -1,3 +1,70 @@
 # Shabad OS Actions
 
-Shabad OS cross-repository actions
+Shabad OS cross-repository actions.
+
+## Usage
+
+Actions are all stored in subfolders.
+
+```yaml
+# example-workflow.yml
+
+name: CI
+
+on: [push]
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: shabados/actions/bump-version@release
+```
+
+### Actions
+
+#### bump-version
+
+Bumps the version of a repository according to the [project management release rules](https://github.com/shabados/.github/wiki/Project-Management#41-release-process).
+
+Achieves this by bumping package.json, and creating a commit, and git tag.
+
+##### Inputs
+
+###### `path`
```

Comment:
I think i'll remove it from this readme and put it into the action's readme, since GitHub action READMEs are supposed to be like: https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-javascript-action#creating-a-readme as far as I know

Reply:
And btw, I was fully conscious of my wrongdoings 😂